BillTrack.Emailer/Function.cs

```diff
@@ -0,0 +1,74 @@
+using System.Net;
+using System.Net.Mail;
+using Amazon.Lambda.Core;
+using Amazon.Lambda.SQSEvents;
+using Amazon.S3;
+using BillTrack.Core.Contracts.SqsMessages;
+using BillTrack.Core.Interfaces.Services;
+using BillTrack.Core.Models;
+using BillTrack.Emailer.Configurations;
+using FluentEmail.Core;
+using FluentEmail.Smtp;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+
+// Assembly attribute to enable the Lambda function's JSON input to be converted into a .NET class.
+[assembly: LambdaSerializer(typeof(Amazon.Lambda.Serialization.SystemTextJson.DefaultLambdaJsonSerializer))]
+
+namespace BillTrack.Emailer;
+
+public class Function
+{
+    private readonly IMessageProcessor _messageProcessor;
+    
+    public Function()
+    {
+        var serviceCollection = new ServiceCollection();
+        ConfigureServices(serviceCollection);
+        var serviceProvider = serviceCollection.BuildServiceProvider();
+
+        _messageProcessor = serviceProvider.GetRequiredService<IMessageProcessor>();
+    }
+    
+    public async Task FunctionHandler(SQSEvent evnt)
+    {
+        foreach (var message in evnt.Records)
```

Comment:
this is the place where the changed `ConcurrentDictionary` could be important.
Why?